src/core/api/thread_api.cpp

```diff
@@ -320,7 +320,7 @@ otError otThreadGetParentInfo(otInstance *aInstance, otRouterInfo *aParentInfo)
     aParentInfo->mPathCost       = parent->GetCost();
     aParentInfo->mLinkQualityIn  = parent->GetLinkInfo().GetLinkQuality();
     aParentInfo->mLinkQualityOut = parent->GetLinkQualityOut();
-    aParentInfo->mAge       = static_cast<uint8_t>(TimerMilli::MsecToSec(TimerMilli::Elapsed(parent->GetLastHeard())));
+    aParentInfo->mAge       = static_cast<uint8_t>(TimeMilli::MsecToSec(TimeMilli::GetNow() - parent->GetLastHeard()));
```

Comment:
nit: any reason not to use `Time::MsecToSec`? `MsecToSec` seems to be defined as a part of `Time`.

Reply:
Good point! Will change them.